src/EventsPage/components/AllUpcomingEvents.js

```diff
@@ -6,6 +6,10 @@ import _ from 'lodash';
 
 const sortByDate = (events = []) => (_.orderBy(events, 'date', 'desc'))
 
+let today = new Date();
+today = today.toLocaleString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit'})
```

Comment:
I'd move these two lines into the `renderEventList` function where we're using them.